build/rpm/pcp.spec.in

```diff
@@ -1858,7 +1858,7 @@ Requires: python3-pcp python3-lxml
 %if "%{_vendor}" == "suse"
 Requires: python3-libvirt-python
 %else
-%if 0%{?centos_ver} == 7
+%if 0%{?centos_ver}%{?rhel} == 7
```

Comment:
@Tualua @kmcdonell can we remove centos_ver here then?  We don't use that rpm variable anywhere else, so it looks a little odd - and even more odd with this change :)